docs/analyzer/analyzer_engine_provider.md

```diff
@@ -0,0 +1,137 @@
+# Configuring the Analyzer Engine from file
+
+Presidio uses `AnalyzerEngineProvider` to load `AnalyzerEngine` configuration from file. 
+Configuration can be loaded in three different ways:
+
+## Using a single file
+
+Create an `AnalyzerEngineProvider` using a single configuration file and set its path to `analyzer_engine_conf_file`, then create `AnalyzerEngine` based on it:
+
+```python
+from presidio_analyzer import AnalyzerEngine, AnalyzerEngineProvider
+
+analyzer_conf_file = "./analyzer/analyzer-config-all.yml"
+
+provider = AnalyzerEngineProvider(
+    analyzer_engine_conf_file=analyzer_conf_file
+    )
+analyzer = provider.create_engine()
+
+results = analyzer.analyze(text="My name is Morris", language="en")
+print(results)
+```
+
+An example configuration file:
+
+```yaml
+supported_languages: 
+- en
+default_score_threshold: 0
+
+nlp_configuration:
+  nlp_engine_name: spacy
+  models:
+  -
+    lang_code: en
```

Comment:
The yaml is still invalid. Consider taking this one: https://github.com/microsoft/presidio/blob/e65c89c68797d1609c7cba176e55450cc635d764/docs/analyzer/nlp_engines/transformers.md?plain=1#L67C1-L99C7